File: docs/user/en/query/filtering/examples/logical/not-narrowed.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityPrimaryKeyInSet(110513, 66567, 106742, 66574, 66556, 110066),
					not(
						entityPrimaryKeyInSet(110066, 106742, 110513)
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/logical/not-narrowed.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityPrimaryKeyInSet(110513, 66567, 106742, 66574, 66556, 110066),
        		Not(
        			EntityPrimaryKeyInSet(110066, 106742, 110513)
        		)
        	)
        )
	)
);